src/components/counter.js

```diff
@@ -7,6 +7,8 @@ import { createStore } from 'redux';
 import { Provider, connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
+document.addEventListener('syncTick', e => console.log(e));
+
```

Comment:
Redux logger will show you all actions that will be triggered. Is this listener needed?